linalg/solvers.hpp

```diff
@@ -473,6 +473,107 @@ class SLBQPOptimizer : public OptimizationSolver
    virtual void Mult(const Vector &xt, Vector &x) const;
 };
 
+/** Block ILU solver:
+ *  Performs a block ILU(k) approximate factorization with specificed block
```

Comment:
The Doxygen style you're using here is not exactly consistent with what's used elsewhere in MFEM (ie, leading `*` on each line). Otherwise, this is excellent and very clear documentation!

Reply:
Typo: specificed

Reply:
Fixed.